crates/driver/src/tests/setup/mod.rs

```diff
@@ -301,6 +327,11 @@ pub struct Solver {
     /// Whether or not solver is allowed to combine multiple solutions into a
     /// new one.
     merge_solutions: bool,
+    /// Whether or not the solver is a JIT-order surplus capturing address
+    surplus_capturing_jit_order_owner: bool,
```

Comment:
I'm not really sure that the solver needs to know whether it's allowed to get surplus.